features/building.feature

```diff
@@ -63,4 +63,16 @@ Feature: Building images
     When a successfully built app at "image"
     Then the file "/cache/images/fox-opt.jpg" should not have been updated
 
-
+  Scenario: Building all images in a custom cache directory
+    Given a fixture app "image"
+    And our extension is enabled with:
+      """
+      config.cache_dir = 'custom/cache'
+      """
+    And a template named "index.html.erb" with:
+      """
+      <%= image_tag 'images/fox.jpg', optimize: true %>
+      """
+    And the Server is running
+    When I go to "/index.html"
```

Comment:
Wherever possible, we should use a test scenario where the app actually gets built. I think this should be possible here by replacing these lines with:

```
    When a successfully built app at "image"
```